SR2025/2024-10-30-under-way.md

```diff
@@ -0,0 +1,99 @@
+---
+to: SR2025 teams
+subject: SR2025 Foundations & Support
+---
+
+Hi,
+
+We hope you enjoyed Kickstart earlier in this month, whether you attended in
+person or [watched the livestream][livestream].
+
+## Getting Started
+
+If your team haven't already done so, the place we'd recommend they start is
+with the [pre-Kickstart activities][pre-kickstart-activities] and the
+[microgames]. Many of the tasks are self-evident when they're completed, however
+for some you may need to refer to the [super-secret answer sheet][microgames-answers].
+
+Together these introduce your team to our [documentation][docs] and
+[Discord][discord], which are key places to get information and support for
+building their robot, as well as giving your team the basic knowledge to get
+started with the kits and their robot.
+
+If after completing the microgames your team are looking for the next step, a
+good place to go would be the [Challenges][challenges]. These are the first
+opportunity to earn points in the competition and start teams building
+components which will be useful for their robot, while still being fairly
+self-contained.
+
+Alongside the Challenges your team may find it useful to read through our
+"[Robots 101][robots-101]" series, which provide general guidance on starting
+working on a robot.
+
+Finally, for the first Tech Day of the year (in [Horsham in late November][horsham-tech-day])
+we will be focussing on supporting your team towards having a moving base for
+their robot. We'll have lots of volunteers on hand with lots of related
+experience, including a mix of ex-competitors willing to share what they learned
+while competing.
+If this interests your team, please **[sign up for the Tech Day][tech-day-signup]**
+as spaces are limited.
```

Comment:
Can we move this to be the first paragraph in the section to encourage signups?

Reply:
How about 89b17520cb37c75fe2ba4bce01fa61ff9d31ce75?